### ISSUE TITLE
Cant install packages on debian

### ISSUE THREAD

**Issue by ketan** [gocd/gocd#3126]
##### Issue Type

- Bug Report

The [installers](https://github.com/gocd/gocd/blob/6fa3a2013a84397af0eec169a275b907297a50bd/buildSrc/src/main/groovy/com/thoughtworks/go/build/DebPackagingTask.groovy#L36) continue to depend on java7-runtime-headless.

For certain versions of ubuntu/debian, Java 8 is not available on the central ubuntu and debian repositories, and require a very specific PPA to be enabled.

**Comment by mathieulj** [comment]
I've just recently encountered this issue. I have been running version 16.12 no problems but I tried to update to 17.1 and the install fails with `Error starting Go Server.`. The only populated log is go-server.out.log and it contains : 
```
[Fri Jan 27 16:07:50 EST 2017] Starting Go Server with command: java -server -Djava.security.egd=file:/dev/./urandom -Xms512m -Xmx1024m -XX:PermSize=128m -XX:MaxMetaspaceSize=256m -Duser.language=en -Djruby.rack.request.size.threshold.bytes=30000000 -Duser.country=US -Dcruise.config.dir=/etc/go -Dcruise.config.file=/etc/go/cruise-config.xml -Dcruise.server.port=8153 -Dcruise.server.ssl.port=8154 -jar /usr/share/go-server/go.jar
[Fri Jan 27 16:07:50 EST 2017] Starting Go Server in directory: /var/lib/go-server
Unrecognized VM option 'MaxMetaspaceSize=256m'
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.
```

I see that the change was mad in commit a2951dfc75bf15d68a049be653de739df20, conveniently title java 8 change. Is java 7 no longer supported? If not it probably would probably be a good idea to update the dependency and the documentation (both of which still mention java 7 and not 8).

**Comment by ketan** [comment]
> I see that the change was made in commit a2951df, conveniently titled java 8 change. Is java 7 no longer supported? If not it probably would be a good idea to update the dependency and the documentation (both of which still mention java 7)*.

Yes, I realized yesterday that our `.deb` package continues to depend on java 7. This is going to be addressed in 17.2. In the meanwhile, I recommend installing `java8-runtime-headless` from the `openjdk-r/ppa` —

```bash
sudo add-apt-repository ppa:openjdk-r/ppa
sudo apt-get update
sudo apt-get install -y openjdk-8-jre
```

I'll updating the documentation accordingly.
